src/groupByKey.ts

```diff
@@ -2,6 +2,22 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
+export function groupByKey(
+  items: [{ [key: string | number]: string }],
+  key: keyof GroupsMap<{ [key: string | number]: string }>,
+): object {
```

Comment:
Please use Generic type in this task and avoid using `object` - it's the same as `any`